packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -936,6 +945,29 @@ export const NewMentionNotification = createNotificationType({
   },
 })
 
+export const NewPingbackNotification = createNotificationType({
+  name: "newPingback",
+  userSettingField: "notificationNewPingback",
+  async getMessage({documentType, documentId, extraData, context}: GetMessageProps) {
+    const summary = await getDocumentSummary(documentType, documentId, context)
+    const prefix = documentType === "comment" ? "their comment on " : "";
+    return `${summary?.associatedUserName} mentioned your ${extraData?.pingbackType} in ${prefix}${summary?.displayName}`
+  },
+  getIcon() {
+    return <CommentsIcon style={iconStyles}/>
+  },
+  Display: ({User, Comment, Post, Tag, notification: {comment, tag, extraData}}) => {
+    if (tag) {
+      return (
+        <><User /> mentioned your ${extraData?.pingbackType} in <Tag /></>
+      );
+    }
+    return (
+      <><User /> mentioned your ${extraData?.pingbackType} in {comment ? <>their <Comment /> on </> : ""}<Post /></>
```

Comment:
Could you change this to "X mentioned your post "title of post"" or (ideally, less important) "X mentioned your comment "start of comment...""?

I think people will be more interested in which of their posts has been referenced than in the content referencing it, plus there isn't really enough room in the notification to read the full string currently.

Sorry I know this will be a pain because it changes the document being referenced but I think it will make a big difference to how much people like getting the notifications.

Reply:
Yep I think that's the right decision 👍